db/migrate/20250325204128_add_index_to_claims_api_power_of_attorneys.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexToClaimsApiPowerOfAttorneys < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
+
+  def change
+    add_index :claims_api_power_of_attorneys, :form_data_hash, algorithm: :concurrently
```

Comment:
Just making sure that we want to index on the `form_data_hash` and not on `header_hash` - the previous md5 index was on `header_md5`.